multisig/simulations/vip-002/vip-002-ethereum/simulations.ts

```diff
@@ -0,0 +1,569 @@
+import { expect } from "chai";
+import { BigNumberish } from "ethers";
+import { Contract } from "ethers";
+import { parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+
+import { forking, pretendExecutingVip } from "../../../../src/vip-framework";
+import { checkVToken } from "../../../../src/vip-framework/checks/checkVToken";
+import { checkInterestRate } from "../../../../src/vip-framework/checks/interestRateModel";
+import { vip002 } from "../../../proposals/vip-002/vip-002-ethereum";
+import COMPTROLLER_ABI from "./abi/comptroller.json";
+import ERC20_ABI from "./abi/erc20.json";
+import POOL_REGISTRY_ABI from "./abi/poolRegistry.json";
+import RATE_MODEL_ABI from "./abi/rateModel.json";
+import VTOKEN_ABI from "./abi/vToken.json";
+
+const RESILIENT_ORACLE = "0xd2ce3fb018805ef92b8C5976cb31F84b4E295F94";
```

Comment:
I would recommend hardcoding as little as possible in the simulation and instead exporting values from the VIP and using them in the simulation. This is an easy way to ensure the simulation is using the same values as the VIP